lib/SymbolGraphGen/SymbolGraph.cpp

```diff
@@ -75,30 +75,39 @@ PrintOptions SymbolGraph::getDeclarationFragmentsPrintOptions() const {
 
   llvm::StringMap<AnyAttrKind> ExcludeAttrs;
 
-#define TYPE_ATTR(X, C) ExcludeAttrs.insert(std::make_pair("TAK_" #X, TAK_##X));
-#include "swift/AST/Attr.def"
+#define TYPE_ATTR(X, C)                                                        \
+  ExcludeAttrs.insert(std::make_pair("TypeAttrKind::" #C, TypeAttrKind::C));
```

Comment:
This string is currently only used as keys of the temporary map. Changing the string value doesn't affect the functionality.